app/src/main/java/com/conupods/MapsActivity.java

```diff
@@ -11,10 +13,15 @@
 import com.google.android.gms.maps.model.LatLng;
 import com.google.android.gms.maps.model.MarkerOptions;
 
-public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
+public class MapsActivity extends FragmentActivity implements OnMapReadyCallback, View.OnClickListener {
```

Comment:
Instead of turning the MapActivity into  an onclickListener as well, I would recommend using lambda/anonymous inner classes instead since the way you are currently doing it tightly couples the map activity and turns it into an onClickListener

Reply:
Done

Reply:
Has some issues getting it to run initially because the default configuration of the project does not support lambda expressions. Since we are using Java 8 it's not a problem though.